docs/docs/features/request-validation.md

```
@@ -86,6 +87,10 @@ type MyStruct2 struct {
 
 Nullable types will generate a type array like `"type": ["string", "null"]` which has broad compatibility and is easy to downgrade to OpenAPI 3.0. Also keep in mind you can always provide a [custom schema](./schema-customization.md) if the built-in features aren't exactly what you need.
 
+!!! info "Note"
+
+    Slices in Go marshal into JSON as `null` if the slice itself is `nil` rather than allocated but empty. This is why slices are nullable by default. See the [Go JSON package documentation](https://pkg.go.dev/encoding/json#Marshal) for more information.
```

Markdownlint

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)